### PR TITLE
fix(docs-infra): set `th`/`td` to proper width

### PR DESCRIPTION
**Pull request by sonukapoor**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

image

## What is the new behavior?

image

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/src/styles/2-modules/_table.scss [hunk 1]
```diff
@@ -113,3 +113,15 @@ table {
     }
   }
 }
+
+article[class="events-container"] {
+  & table > tbody >
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_table.scss]: This seems redundant. Please remove.

File: aio/src/styles/2-modules/_table.scss [hunk 2]
```diff
@@ -113,3 +113,15 @@ table {
     }
   }
 }
+
+article[class="events-container"] {
+  & table > tbody > 
+  
+  tr > th:first-child {
+    width: 15%;
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_table.scss]: On narrow screens, this makes the first column too narrow while other columns has empty space. How about making each column 33% wide?

**Review by gkalpak — CHANGES_REQUESTED** [review]
Good catch!

File: aio/src/styles/2-modules/_table.scss [hunk 3]
```diff
@@ -113,3 +113,15 @@ table {
     }
   }
 }
+
+article[class="events-container"] {
```
[the remark this answers is not in the document]
Comment (sonukapoor) [reply aio/src/styles/2-modules/_table.scss]: Haha. What was I thinking :)

File: aio/src/styles/2-modules/_table.scss (hunk 2 above)
Comment (sonukapoor) [reply aio/src/styles/2-modules/_table.scss]: Makes sense.

File: aio/src/styles/2-modules/_table.scss [hunk 4]
```diff
@@ -114,14 +114,13 @@ table {
...
   
-  tr > th:first-child {
-    width: 15%;
+  tr > th {
+    width: 33%;
   }
 
-  tr > td:last-child {
-    width: 30%;
+  tr > td {
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_table.scss]: Let's combine these rules to one:

```scss
.events-container {
   tr > td, tr > th {
    width: 33%;
  }
}
```

**Review by gkalpak — CHANGES_REQUESTED** [review]
Thx for making the changes, @sonukapoor! Can you also change the commit message type/scope to `fix(docs-infra): ...`?

File: aio/src/styles/2-modules/_table.scss [hunk 5]
```diff
@@ -113,3 +113,9 @@ table {
     }
   }
 }
+
+.events-container {
+  tr > td, tr > th {
+   width: 33%;
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_table.scss]:
```suggestion
    width: 33%;
```

File: aio/src/styles/2-modules/_table.scss [hunk 6]
```diff
@@ -113,3 +113,9 @@ table {
...
+ }
```
Comment (gkalpak) [inline aio/src/styles/2-modules/_table.scss]:
```suggestion
  }
```

**Review by gkalpak — CHANGES_REQUESTED** [review]

**Review by gkalpak — APPROVED** [review]
👌

**Pull request closed without merging**